src/hope_interpreter.cpp

```diff
@@ -94,6 +94,12 @@ void Interpreter::interpretStmt(ParseNode pn){
         case OP_CONTINUE: status = static_cast<Status>(status | CONTINUE); break;
         case OP_PLOT: plotStmt(pn); break;
         case OP_DO_NOTHING: break;
+        case OP_NAMESPACE:{
+            stack.push(static_cast<void*>(nullptr)
+                DEBUG_STACK_ARG("namespace-" + parse_tree.str(parse_tree.lhs(pn))));
```

Comment:
Bad. You've assumed every symbol has a role in the runtime, and that's a bad assumption for multiple reasons.